recipes/knoten/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "knoten" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/knoten/archive/{{ version }}.tar.gz
+  sha256: d671f1c8c32107cf1ef1d2dadf236ba98f9f19dcfb744e302cb20860748164b0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [ win or py2k ]
```

Comment:
This package is pure python. It should be `noarch: python`.
Please remove the `skip`. It doesn't matter if the package can't be installed on windows due to missing dependencies. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python